priv/perf/apps/load_test/lib/test_runner/config.ex

```diff
@@ -0,0 +1,60 @@
+# Copyright 2019-2020 OmiseGO Pte Ltd
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+defmodule LoadTest.TestRunner.Config do
+  @moduledoc """
+  Command line args parser for TestRunner.
+  """
+  alias ExPlasma.Encoding
+
+  @tests %{
+    "deposits" => LoadTest.Runner.Deposits
+  }
+
+  @configs %{
+    "deposits" => %{
+      token: Encoding.to_binary("0x0000000000000000000000000000000000000000"),
+      initial_amount: 500_000_000_000_000_000,
+      deposited_amount: 200_000_000_000_000_000,
+      transferred_amount: 100_000_000_000_000_000
+    }
+  }
+  def parse() do
+    [test, rate, period] = System.argv()
+
+    config = config(test, rate, period)
+
+    runner_module = Map.fetch!(@tests, test)
+
+    {runner_module, config}
```

Comment:
should we start a document in describing how to actually run this?

also, I think there was a concern chaperon would blow up if huge parameters would be used? should we validate inputs?

Reply:
> should we start a document in describing how to actually run this?

I added moduledoc to the `TestRunner` module

> also, I think there was a concern chaperon would blow up if huge parameters would be used? should we validate inputs?

I added validation. if it spawns more than 200_000 processes, an error will be raised